evm/src/executor/backend/mod.rs

```diff
@@ -42,8 +41,7 @@ pub type LocalForkId = U256;
 type ForkLookupIndex = usize;
 
 /// All accounts that will have persistent storage across fork swaps. See also [`clone_data()`]
-const DEFAULT_PERSISTENT_ACCOUNTS: [H160; 3] =
-    [CALLER, CHEATCODE_ADDRESS, DEFAULT_CREATE2_DEPLOYER];
+const DEFAULT_PERSISTENT_ACCOUNTS: [H160; 2] = [CHEATCODE_ADDRESS, DEFAULT_CREATE2_DEPLOYER];
```

Comment:
unsure if really necessary, but don't see why not either

Reply:
cheatcode is necessary since `failed` needs to be persisted across swaps, idk about create2 deployer

Reply:
i was referring to removing `CALLER`, but that's good to know